package.json

```diff
@@ -8,7 +8,9 @@
     "start": "next start",
     "lint": "next lint",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
```

Comment:
you removed the storybook files from your project, you can also remove the entries from you package.json and run `npm install` afterwards to ensure the changes are applied